core/client/app/templates/setup/two.hbs

```diff
@@ -46,8 +45,5 @@
         {{/gh-form-group}}
     </form>
 
-    <button type="submit" class="btn btn-green btn-lg btn-block" {{action "setup"}} {{submitting}}>Last step: Invite your team <i class="icon-chevron"></i></button>
-    {{#if showError}}
-    <p class="main-error">{{invalidMessage}}</p>
-    {{/if}}
-</section>
+<button type="submit" class="btn btn-green btn-lg btn-block" {{action "setup"}} {{submitting}}>Last step: Invite your team <i class="icon-chevron"></i></button>
```

Comment:
I'm not really sure I felt like perhaps it was meant for something else, like adding a class but I couldn't figure it out which is why I left it in. I'll have another play with it when I update with all these suggestions. Perhaps i ought to take it out. (if you can't see what it is doing then it probably isn't doing anything)